src/main/java/com/ontotext/kafka/producer/GraphDBProducer.java

```diff
@@ -117,7 +111,7 @@ public void publish() {
                                 keyName = keyLine.split("=")[1];
                                 keyStatements = keysReader.readLine().split("=")[1];
                                 numberOfStatementsPerKey = Integer.parseInt(keyStatements);
-                                countStatementsInCurrentMessage = 0;
+								countStatementsInCurrentMessage = 0;
```

Comment:
Formatting broke here

Reply:
This is pretty strange diff - it looks OK in my IDE
![image](https://github.com/user-attachments/assets/04fa8576-e6ca-45c8-9796-ff85488ee625)